protected/humhub/modules/user/Module.php

```diff
@@ -236,4 +236,14 @@ public function setDefaultGroup($id)
             $group->save();
         }
     }
+
+    /**
+     * Check the blocking users is allowed
+     *
+     * @return bool
+     */
+    public function allowBlockUsers(): bool
+    {
+        return (bool) $this->settings->get('auth.blockUsers', false);
```

Comment:
@yurabakhtin The setting should be activated by default. Possibly in migration?

Reply:
@luke- No need migration, I just set `true` as default value - https://github.com/humhub/humhub/pull/5309/commits/0718c2102d30b5d3674df1358b3a0ef944410028.